app/feature/feature-movingflow/src/main/graphql/MutationMoveIntentRequest.graphql

```diff
@@ -0,0 +1,62 @@
+mutation MoveIntentV2Request($intentId: ID!, $moveIntentRequestInput: MoveIntentRequestInput!) {
+  moveIntentRequest(intentId: $intentId, input: $moveIntentRequestInput) {
+    moveIntent {
+      ...MoveIntentQuotesFragment
+    }
+    userError {
+      message
+    }
+  }
+}
+
+fragment MoveIntentQuotesFragment on MoveIntent {
+  homeQuotes {
+    id
+    premium {
+      ...MoneyFragment
+    }
+    startDate
+    defaultChoice
+    tierName
+    tierLevel
+    deductible {
+      amount {
+        ...MoneyFragment
+      }
+      percentage
+      displayText
+    }
+    displayItems {
+      ...MoveQuoteDisplayItemFragment
+    }
+    exposureName
+    productVariant {
+      ...ProductVariantFragment
+      # These are not in the fragment, why?
+      #          termsVersion
+      #          displayNameSubtype
+    }
+  }
+  mtaQuotes {
+    premium {
+      ...MoneyFragment
+    }
+    startDate
+    displayItems {
+      ...MoveQuoteDisplayItemFragment
+    }
+    exposureName
+    productVariant {
+      ...ProductVariantFragment
+      # These are not in the fragment, why?
+      #          termsVersion
+      #          displayNameSubtype
```

Comment:
I believe I've added termsVersion in here: https://github.com/HedvigInsurance/android/pull/2257 (comparison api) but it's still a draft

Reply:
Yeah, I'll let this be for now while comparison stuff is still WIP, no need to do anything about it for now